bin/varnishtest/tests/v00019.vtc

```diff
@@ -109,6 +109,17 @@ varnish v1 -vcl {
 	}
 }
 
+# regression test #4146
+varnish v1 -vcl {
+	import debug;
+	backend be none;
+	sub vcl_init {
```

Comment:
Now that we actually run `vcl_init` in these tests, can you please edit the existing tests where vcl compilation is expected to succeed (replace `vcl_recv` by `vcl_init` and adapt the conditions accordingly) ? This will ensure that the generated code is correct and produces expected results.